pom.xml

```diff
@@ -426,7 +426,7 @@
 
 					<!-- Third argument prevents Java from popping up lots of windows on
 						MacOS -->
```

Comment:
@deroneriksson, @frreiss, @gweidner, @dusenberrymw - I've increased the maximum memory needed by the JVM from 2G to 8G. Is this ok?